stripe/src/main/java/com/stripe/android/util/StripeNetworkUtils.java

```diff
@@ -37,13 +41,41 @@
         cardParams.put("address_country", StripeTextUtils.nullIfBlank(card.getAddressCountry()));
 
         // Remove all null values; they cause validation errors
-        for (String key : new HashSet<>(cardParams.keySet())) {
-            if (cardParams.get(key) == null) {
-                cardParams.remove(key);
-            }
-        }
+        removeNullParams(cardParams);
 
         tokenParams.put("card", cardParams);
         return tokenParams;
     }
+
+    @NonNull
+    public static Map<String, Object> hashMapFromBankAccount(@NonNull BankAccount bankAccount) {
+        Map<String, Object> tokenParams = new HashMap<>();
+        Map<String, Object> accountParams = new HashMap<>();
+
+        accountParams.put("country", bankAccount.getCountryCode());
+        accountParams.put("currency", bankAccount.getCurrency());
+        accountParams.put("account_number", bankAccount.getAccountNumber());
+        accountParams.put("routing_number",
+                StripeTextUtils.nullIfBlank(bankAccount.getRoutingNumber()));
+        accountParams.put("account_holder_name",
+                StripeTextUtils.nullIfBlank(bankAccount.getAccountHolderName()));
+        accountParams.put("account_holder_type",
+                StripeTextUtils.nullIfBlank(bankAccount.getAccountHolderType()));
+
+        // Remove all null values; they cause validation errors
+        removeNullParams(accountParams);
+
+        tokenParams.put("bank_account", accountParams);
+        return tokenParams;
+    }
+
+    static void removeNullParams(Map<String, Object> mapToEdit) {
+        // Remove all null values; they cause validation errors
+        for (String key : new HashSet<>(mapToEdit.keySet())) {
```

Comment:
It would be cool to use this, but in order to do so I need to install the Jack compiler, which I haven't yet done. I could theoretically take the plunge on that, but it would actually close off some options that I might want (like certain types of lint and the ability to use certain code coverage tools like JaCoCo). I don't think it disables anything I'm using right now, but since it's [not entirely back-compatible](https://developer.android.com/guide/platform/j8-jack.html), I'm inclined not to go for it right now.